cluster/plan.go

```diff
@@ -691,6 +691,7 @@ func (c *Cluster) BuildSidecarProcess() v3.Process {
 		Labels: map[string]string{
 			ContainerNameLabel: services.SidekickContainerName,
 		},
+		Command: []string{"/bin/bash"},
```

Comment:
is it to address some different issue?

Reply:
Confirmed: it is to address some slightly different issue when the upgrade was *always* performed for the sidekick container as on rke side the command verification was done incorrectly. And it contributed to the mass of the volumes being left around.